src/libstd/error.rs

```diff
@@ -119,6 +119,15 @@ impl From<String> for Box<Error + Send + Sync> {
     }
 }
 
+#[unstable(feature = "string_box_error", reason = "recently added", issue = "30156")]
```

Comment:
Implementations aren't checked for stability so this should really be flagged stable.